terraform/stacks/networking/main.tf

```diff
@@ -103,3 +103,52 @@ module "main_vpc" {
     Creator     = "terraform"
   }
 }
+
+resource "aws_security_group" "main_vpc_sg_outbound" {
+  name        = "main-vpc-sg-outbound"
+  description = "Main VPC Security Group allow outbound only traffic"
+  vpc_id      = module.main_vpc.vpc_id
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name        = "outbound"
+    Tier        = var.umccr_subnet_tier.PRIVATE
+    Environment = terraform.workspace
+    Stack       = var.stack_name
+    Creator     = "terraform"
+  }
+}
+
+resource "aws_security_group" "main_vpc_sg_uom" {
+  name        = "main-vpc-sg-uom"
+  description = "Main VPC Security Group allow outbound traffic and UoM only inbound to SSH"
+  vpc_id      = module.main_vpc.vpc_id
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["10.1.0.0/20"]
+  }
+
+  tags = {
+    Name        = "uom"
```

Comment:
I'd use ssh_from_uom, again to be clearer

Reply:
Amended